include/cantera/numerics/Integrator.h

```diff
@@ -92,7 +93,8 @@ class Integrator
     /*!
      * @param probtype    Type of the problem
      */
-    virtual void setProblemType(int probtype) {
+    virtual void setProblemType(int probtype) 
+    {
```

Comment:
I'm not sure what changed here, maybe line endings? Can you make sure you're using only "LF" line endings, not "CRLF"?